docs/overview/code-editor-environment.md

```diff
@@ -27,9 +27,15 @@ Office Scripts does not support the usage of external, third-party JavaScript li
 
 ## IntelliSense
 
-IntelliSense is a Code Editor feature that helps prevent typos and syntax errors as you edit your script. It displays possible object and field names as you type, as well as inline documentation for every API.
+IntelliSense is a set of Code Editor features that help you write code. It provides auto-complete, syntax error highlighting, and inline API documentation.
 
-The Excel Code Editor uses the same IntelliSense engine as Visual Studio Code. To learn more about the feature, visit [Visual Studio Code's IntelliSense Features](https://code.visualstudio.com/docs/editor/intellisense#_intellisense-features).
+IntelliSense gives suggestions as you type, similar to the suggested text in Excel. Pressing **Tab** or **Enter** inserts the suggested member. Trigger IntelliSense at the current cursor location by pressing the Ctrl+Space. These suggestions are especially useful when completing a method. The displayed method signature contains a list of arguments it needs, each argument's type, whether a given argument is required or optional, and the return type of the method.
```

Comment:
Question: 
Should **Tab** and **Enter** be formatted like keys, and not like UI buttons? I think UI buttons are formatted as bold text in our docs, and keys are referred to as +"key" with no additional formatting. 

Other misc. updates: 
Added a missing word. 
A slight clarification to the final sentence. 

```suggestion
IntelliSense gives suggestions as you type, similar to the suggested text in Excel. Pressing the Tab or Enter key inserts the suggested member. Trigger IntelliSense at the current cursor location by pressing the Ctrl+Space keys. These suggestions are especially useful when completing a method. The method signature displayed by IntelliSense contains a list of arguments it needs, each argument's type, whether a given argument is required or optional, and the return type of the method.
```

Reply:
You're right, the keyboard buttons should not be formatted like that. Good call.